config/locales/es/activerecord.yml

```diff
@@ -315,6 +318,11 @@ es:
         code: "Código"
         title: "Título"
         description: "Descripción"
+      sdg/local_target:
+        code: "Código"
+      sdg/local_target/translation:
+        title: "Título"
```

Comment:
In the index table, the text "Title" is present in Spanish, because (unfortunately :relieved:), I think we need to add a translation both to `sdg/local_target/translation` and `sdg/local_traget`.

Reply:
Done.